source/includes/fundamentals/code-snippets/CompoundOperators.java

```diff
@@ -137,13 +158,22 @@ class DemoClientSafe extends DemoClient {
 
     // start the-safe-book-a-room
     public void bookARoom(){
+        // Creates an update document to mark a room as reserved
         Bson update = Updates.combine(Updates.set("reserved", true), Updates.set("guest", guest));
+        
+        // Creates a filter to match a document representing an available room
         Bson filter = Filters.eq("reserved", false);
+
+        // Atomically finds and updates a document to mark it as reserved
```

Comment:
Suggestion:

I think we landed on the following for findOneAndDelete which should be applied to findOneAndUpdate in the [code comments sheet](https://docs.google.com/spreadsheets/d/1gCIeV5Bsp1G69MUNQZm8bYK9UNVSQjJt9T7eUM8aJiE/edit#gid=0):

```suggestion
        // Updates the first document that matches the filter to mark it as reserved
```

Applies to https://github.com/mongodb/docs-java/pull/472/files#diff-3d6441156a561267d83bde841144e80a788371a97373d78a95825cb1d584c919R118